blocks/api/test/function-refs.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Being that this file is placed in the `test` directory, Mocha will try to read and interpret it as a test suite, which it's not. Perhaps we could either inline these functions where they're used, or create a nested `fixtures` directory?